README.md

```diff
@@ -1,19 +1,17 @@
-<img src="https://spec.edmcouncil.org/fibo/htmlpages/master/latest/img/logo.66a988fe.png" width="150" align="right"/>
+<img src="https://spec.edmcouncil.org/fibo/htmlpages/master/develop/img/logo.66a988fe.png" width="150" align="right"/>
```

Comment:
Should be:
`https://spec.edmcouncil.org/fibo/htmlpages/develop/latest/img/logo.66a988fe.png`